### PR TITLE
Fix disabled scrolling on mobile

### LINKED ISSUE #4556: Found scrolling disable issue
### What happened?

I discovered an issue where scrolling was incorrectly disabled when the screen size was mobile.

Let's start on Homepage.
1. Tap the nav menu icon (at mobile screen) at the header
2. And then click Create/Classic
3. https://kodadot.xyz/bsx/create You can see this page. Can you scrolling to the bottom? Maby No!!!

You can see this kind of problem everywhere.
Not only when clicking 'Create/Classic', other page clicking also disable the scrolling.
Also even you didn't start on Homepage, you can see scrolling disabled homepage when you click header logo after tap the mobile view nav menu icon.

### Please reproduce in steps

Let's start on Homepage.
1. Tap the nav menu icon (at mobile screen) at the header
2. And then click Create/Classic
3. https://kodadot.xyz/bsx/create You can see this page. Can you scrolling to the bottom? Maby No!!!

### Expected Behavior

In any case, I didn't expect to need to disable scrolling when the content goes over the page.

### What browsers are you seeing the problem on?

Chrome

### At which address did you encounter bug?

kodadot.xyz

### Are you logged in?

No

### Which wallet you are using?

PolkadotJS

### At which chain did you encounter bug?

Basilisk, MoonSama, RMRK

### Screenshots

https://user-images.githubusercontent.com/88679102/209338462-ebbbeeb9-869e-425d-93bb-20ed8c563884.mp4

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/transfer/?target=EipjuPLU8F3435MGCaS972u7wxwhvi3H8KzJ1e95qWEDtKM

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #4556 
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/Navbar.vue [hunk 1]
```diff
@@ -326,11 +319,14 @@ export default class NavbarMenu extends mixins(
...
+    const navbarMenu = document.querySelector('.navbar-menu')
+    return navbarMenu?.classList.contains('is-active')
+  }
+
   @Watch('isBurgerMenuOpened')
   onBurgerMenuOpenedChanged() {
-    document.documentElement.style.overflow = this.isBurgerMenuOpened
-      ? 'hidden'
-      : 'auto'
+    this.toggleBodyScroll()
```
Comment (Jarsen136) [inline components/Navbar.vue:329]: I don't agree to add `toggleBodyScroll` everywhere. We even could not know whether it will turn on the scroll or not.

**Review by Jarsen136 — CHANGES_REQUESTED** [review]
1. IMO, the better approach is to watch the `isMenuOpen` status change. And then control whether it should scroll.

like this way:
```
 toggleBodyScroll(on: boolean) { 
  ...
  } 

this.toggleBodyScroll(true)
this.toggleBodyScroll(false)
```

2. There are two similar status: `isBurgerMenuOpened` and `isMobileMenuOpen`. Could we try to unify them?

File: components/Navbar.vue [hunk 2]
```diff
@@ -326,11 +319,14 @@ export default class NavbarMenu extends mixins(
     clearSession()
   }
 
+  get isMobileMenuOpen() {
+    const navbarMenu = document.querySelector('.navbar-menu')
+    return navbarMenu?.classList.contains('is-active')
+  }
```
Comment (Jarsen136) [inline components/Navbar.vue:325]: The function seems not related to `mobile`. How about naming `isNavbarMenuOpen`?

Comment (daiagi) [reply components/Navbar.vue:325]: it is related to mobile
the class is-active appears only when mobile menu is opened

**Comment by daiagi** [comment]
@yangwao  pinged you on disord

File: components/Navbar.vue [hunk 3]
```diff
@@ -399,6 +389,8 @@ export default class NavbarMenu extends mixins(
 
   beforeDestroy() {
     window.removeEventListener('scroll', this.onScroll)
+    this.setBodyScroll(true)
+    document.documentElement.classList.remove('is-clipped-touch')
```
Comment (Jarsen136) [inline components/Navbar.vue:393]: The last question. Where does the class name `is-clipped-touch` come from?

Comment (daiagi) [reply components/Navbar.vue:393]: B-dropdown
This class is being put on the html element when one of the drop downs is opened
Like the create dropdown for example

And it stays there after click and navigation

Comment (Jarsen136) [reply components/Navbar.vue:393]: Got it.

**Review by Jarsen136 — APPROVED** [review]
code lgtm

**Comment by yangwao** [comment]
> @yangwao pinged you on disord

I haven't seen discord on month+, I was about to write you week ago, it's on my agenda. 
Feel free to write me DM :)

**Comment by yangwao** [comment]
pay 40 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $40 @ 22.37 USD/KSM ~ 1.788 $KSM
🧗 [EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L](https://kusama.subscan.io/account/EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L)
🔗 [0x1ad43452ccb4d4e67572a5bd03bfb17fae0a85efd5b17abd4f81ba0935c65823](https://kusama.subscan.io/extrinsic/0x1ad43452ccb4d4e67572a5bd03bfb17fae0a85efd5b17abd4f81ba0935c65823)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by daiagi** [comment]
@yangwao  I sent you a DM on discord

**Pull request merged**